### PR TITLE
API-49 add endpoint that will return all supported directories

### PR DESCRIPTION
**Pull request by alnarozhny**

### CODE REVIEW

File: source/includes/_appendix.md [hunk 1]
```diff
@@ -30,301 +30,76 @@ One format may work better than another for a specific business so we recommend
...
-yahoo|<span class="yes">Yes</span>||<span class="yes">Yes</span>|<span class="yes">Yes</span>|
-yalwa|<span class="yes">Yes</span>||<span class="yes">Yes</span>||
-yellowee|<span class="yes">Yes</span>||<span class="yes">Yes</span>|<span class="yes">Yes</span>|
-yellowpagesca|<span class="yes">Yes</span>|<span class="yes">Yes</span>|<span class="yes">Yes</span>|<span class="yes">Yes</span>|
-yelp|<span class="yes">Yes</span>|<span class="yes">Yes</span>|<span class="yes">Yes</span>|<span class="yes">Yes</span>|Was previously referred to as yelp-can
-ylm|<span class="yes">Yes</span>||<span class="yes">Yes</span>|<span class="yes">Yes</span>|
-<s>ziplocal</s>|<span class="yes">Yes</span>||<span class="yes">Yes</span>|<span class="yes">Yes</span>|No longer available
-zomato|<span class="yes">Yes</span>|<span class="yes">Yes</span>|<span class="yes">Yes</span>||
-zoominfo|<span class="yes">Yes</span>||<span class="yes">Yes</span>|<span class="yes">Yes</span>|
+We no longer list the supported local directories here. We now have an below which enables you to retrieve a list of directories either all or for an individual country (See list of Supported Countries below).
```
Comment (ejeliot) [inline source/includes/_appendix.md]: We now have an API method (shown below) that enables you to retrieve a list of directories for all supported countries.

File: source/includes/_appendix.md [hunk 2]
```diff
@@ -30,301 +30,76 @@ One format may work better than another for a specific business so we recommend
...
+$success = $api->post('v2/get-directories');
+print_r($success);
+```
+
+```shell
+curl -X POST \
+ -d 'api-key=<INSERT_API_KEY>' \
+ -d 'sig=<INSERT_API_SIG>' \
+ -d 'expires=<INSERT_API_EXPIRES>' \
+ https://tools.brightlocal.com/seo-tools/api/v2/get-directories
```
Comment (ejeliot) [inline source/includes/_appendix.md]: /seo-tools/api/v1/directories/all
/seo-tools/api/v1/directories/\<country\>

**Review by ejeliot — CHANGES_REQUESTED** [review]

**Review by ejeliot — APPROVED** [review]

File: source/includes/_appendix.md [hunk 3]
```diff
@@ -30,301 +30,76 @@ One format may work better than another for a specific business so we recommend
...
+<span class="label label-info">Account Method</span>
+
+> Get list of supported local directories
+
+```php
+<?php
+use BrightLocal\Api;
+
+$api = new Api('<INSERT_API_KEY>', '<INSERT_API_SECRET>');
+$success = $api->post('v1/directories/all');
```
Comment (redhatua) [inline source/includes/_appendix.md]: Should be GET

File: source/includes/_appendix.md [hunk 4]
```diff
@@ -30,301 +30,76 @@ One format may work better than another for a specific business so we recommend
...
+print_r($success);
+```
+
+```shell
+curl -X POST \
```
Comment (redhatua) [inline source/includes/_appendix.md]: Should be GET

File: source/includes/_appendix.md [hunk 5]
```diff
@@ -30,301 +30,76 @@ One format may work better than another for a specific business so we recommend
...
+ -d 'expires=<INSERT_API_EXPIRES>' \
+ https://tools.brightlocal.com/seo-tools/api/v1/directories/all
+```
+
+```csharp
+api request = new api("<INSERT_API_KEY>", "<INSERT_API_SECRET>");
+
+var parameters = new api.Parameters();
+
+var success = request.Post("/v1/directories/all", parameters);
```
Comment (redhatua) [inline source/includes/_appendix.md]: Should be GET

File: source/includes/_appendix.md [hunk 6]
```diff
@@ -30,301 +30,76 @@ One format may work better than another for a specific business so we recommend
...
+		        "supports_reviews": true
+		},...	        
+        ]
+    }
+}
+```
+
+### HTTP Request
+
+`POST https://tools.brightlocal.com/seo-tools/api/v1/directories/all`
```
Comment (redhatua) [inline source/includes/_appendix.md]: Should be GET

File: source/includes/_appendix.md [hunk 7]
```diff
@@ -30,301 +30,76 @@ One format may work better than another for a specific business so we recommend
...
+
+`POST https://tools.brightlocal.com/seo-tools/api/v1/directories/<country>`
```
Comment (redhatua) [inline source/includes/_appendix.md]: Should be GET

**Review by redhatua — CHANGES_REQUESTED** [review]

**Review by redhatua — APPROVED** [review]

**Review by redhatua — APPROVED** [review]

**Pull request merged**